resources/qml/WelcomePages/AddPrinterByIpContent.qml

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the 'by IP' page of the "Add New Printer" flow of the on-boarding process.
+//
+Item
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    id: addPrinterByIpScreen
+
+    property bool hasPushedAdd: false
+    property bool hasSentRequest: false
+    property bool haveConnection: false
+
+    Timer
+    {
+        id: tempTimerButton
+
+        interval: 1200
+        running: false
+        repeat: false
+        onTriggered:
+        {
+            hasPushedAdd = true
+            tempTimerRequest.running = true
+        }
+    }
+    // TODO: Remove timers after review interface!
+
+    Timer
+    {
+        id: tempTimerRequest
+
+        interval: 1200
+        running: false
+        repeat: false
+        onTriggered:
+        {
+            hasSentRequest = true
+            tempTimerConnection.running = true
+        }
+    }
+    // TODO: Remove timers after review interface!
+
+    Timer
+    {
+        id: tempTimerConnection
+
+        interval: 1200
+        running: false
+        repeat: false
+        onTriggered: haveConnection = true
+    }
+    // TODO: Remove timers after review interface!
+
+    Label
+    {
+        id: titleLabel
+        anchors.top: parent.top
+        anchors.topMargin: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        horizontalAlignment: Text.AlignHCenter
+        text: catalog.i18nc("@label", "Add printer by IP adress")
+        color: UM.Theme.getColor("primary_button")
+        font: UM.Theme.getFont("large_bold")
+        renderType: Text.NativeRendering
+    }
+
+    Rectangle
+    {
+        anchors.top: titleLabel.bottom
+        anchors.bottom: connectButton.top
+        anchors.topMargin: 40
+        anchors.bottomMargin: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        width: parent.width * 3 / 4
```

Comment:
@rburema Probably it doesn't look too wide. In the "What's New" page, I did something similar, or the page looks stretched because the text area is too wide.